components/filesystem/lock_handler.rst

```diff
@@ -24,7 +24,7 @@ Usage
 
 A lock can be used, for example, to allow only one instance of a command to run.
 
-.. code-block:: php
+::
```

Comment:
Can we move this to the end of the previous line? Thanks!

Reply:
While reading this comment there is at least one important difference to point out that we did not discuss on Slack: The colon of the short-hand notation will be rendered in the final document. So we need to check too if that really makes sense with each example.